app/components/task/holder.hbs

```diff
@@ -1,16 +1,44 @@
-<div class="task-card">
+<div class="task-card" tabindex="0">
```

Comment:
added `tabindex` so each task card can be highlighted when traversing via keyboard